dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -396,6 +562,141 @@ public void testPrivateBitstream() throws Exception {
+                .withAdminGroup(adminParentCommunity)
+                .build();
+
+        EPerson adminCol1 = EPersonBuilder.createEPerson(context)
+                .withEmail("admin1@mail.com")
+                .withPassword("qwerty01")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity)
+                .withName("Collection 1")
+                .withAdminGroup(adminCol1)
+                .build();
+
+        EPerson adminCol2 = EPersonBuilder.createEPerson(context)
+                .withEmail("admin2@mail.com")
+                .withPassword("qwerty02")
+                .build();
+        Collection col2 = CollectionBuilder.createCollection(context, parentCommunity)
+                .withName("Collection 2")
+                .withAdminGroup(adminCol2)
+                .build();
+
+        Group restrictedGroup = GroupBuilder.createGroup(context)
+                .withName("Restricted Group")
+                .addMember(eperson)
+                .build();
+
+        String bitstreamContent = "Private!";
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+
+            Item item = ItemBuilder.createItem(context, col1)
+                    .withTitle("item")
+                    .withIssueDate("2018-10-17")
+                    .withAuthor("Doe, John")
+                    .build();
+            Bitstream bitstream = BitstreamBuilder
+                .createBitstream(context, item, is)
+                .withName("Test Embargoed Bitstream")
+                .withDescription("This bitstream is embargoed")
+                .withMimeType("text/plain")
+                .withReaderGroup(restrictedGroup)
+                .build();
+
+
+            // download the bitstream
+            // parent community's admin user is allowed access to the item belong restricted group
+            String tokenAdminParentCommuity = getAuthToken(adminParentCommunity.getEmail(), "qwerty00");
+            getClient(tokenAdminParentCommuity).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                                .andExpect(status().isOk());
+
+            // collection1's admin user is allowed access to the item belong restricted group
+            String tokenAdminCol1 = getAuthToken(adminCol1.getEmail(), "qwerty01");
+            getClient(tokenAdminCol1).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                                .andExpect(status().isOk());
+
+            checkNumberOfStatsRecords(bitstream, 2);
+
+            // collection2's admin user is NOT allowed access to the item belong collection1
+            String tokenAdminCol2 = getAuthToken(adminCol2.getEmail(), "qwerty02");
+            getClient(tokenAdminCol2).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                                .andExpect(status().isForbidden());
```

Comment:
It'd be good to again add a second Community that is not a parent of collection1, and ensure that Community's Admin is also Forbidden.